src/hotspot/share/gc/g1/g1CardSet.hpp

```diff
@@ -221,19 +218,19 @@ class G1CardSet : public CHeapObj<mtGCCardSet> {
   static const uintptr_t CardSetHowl           = 0x3;
 
   // The special sentinel values
-  static constexpr CardSetPtr FreeCardSet = nullptr;
-  // Unfortunately we can't make (G1CardSet::CardSetPtr)-1 constexpr because
+  static constexpr ContainerPtr FreeCardSet    = nullptr;
```

Comment:
```suggestion
  static constexpr ContainerPtr FreeCardSet = nullptr;
```
Let's avoid lining up fairly unrelated constants.